doc/source/train/user-guides/persistent-storage.rst

```diff
@@ -1,62 +1,370 @@
-Configuration and Persistent Storage
-====================================
-
 .. _persistent-storage-guide:
 
-.. _train-run-config:
+.. _train-log-dir:
+
+Configuring Persistent Storage
+==============================
+
+A Ray Train run produces results, checkpoints, and other artifacts.
+You can configure these to be saved to a persistent storage location.
+
+.. figure:: ../images/persistent_storage_checkpoint.png
+    :align: center
+    :width: 600px
+
+    An example of multiple workers spread across multiple nodes interacting with persistent storage.
+
+By default, these training outputs are saved in a local directory under ``~/ray_results``.
+This is sufficient for single-node setups or distributed training without saving
+model checkpoints or artifacts, but some form of external persistent storage such as
+cloud storage (e.g., S3, GCS) or NFS (e.g., AWS EFS, Google Filestore) is
+highly recommended when doing multi-node training.
```

Comment:
```suggestion
required when doing multi-node training.
```

Reply:
It's not 100% required if no checkpointing is done, so wasn't sure how to word this